mor/mor/feel/feelmor/crb.hpp

```diff
@@ -5776,13 +5776,13 @@ CRB<TruthModelType>::fixedPoint(  size_type N, parameter_type const& mu, std::ve
     matrix_info_tuple matrix_info;
     tic();
     matrix_info = fixedPointPrimal( N, mu , uN , uNold, output_vector, K , print_rb_matrix, computeOutput ) ;
-    VLOG(3) << fmt::format("CRB::fixedPoint() : fixedPointPrimal took {} seconds",toc("CRB::fixedPoint() : fixedPointPrimal"));
+    LOG(INFO) << fmt::format("CRB::fixedPoint() : fixedPointPrimal took {} seconds", toc("CRB::fixedPoint() : fixedPointPrimal", false));
```

Comment:
please get the output of  toc() outside of LOG or VLOG
and use VLOG because we don't want to have this written all the time 

so
```c++
double t = toc("CRB::fixedPoint()");
VLOG(3) << .... << t << 
```